README.md

```diff
@@ -29,13 +29,13 @@ A Rust library for random number generation, featuring:
 It's also worth pointing out what `rand` *is not*:
 
 -   Small. Most low-level crates are small, but the higher-level `rand` and
-    `rand_distr` each contain a lot of functionality.
--   Simple (implementation). We have a strong focus on correctness, speed and flexibility, but
+    `rand_distr` each contains a lot of functionality.
+-   Simple (implementation). We have a strong focus on correctness, speed, and flexibility, but
```

Comment:
I don't have anything against use of an Oxford comma here, but it's a matter of style rather than correctness.